unlock-app/app/api/svg-to-png/route.tsx

```diff
@@ -0,0 +1,41 @@
+import sharp from 'sharp'
+import { NextRequest, NextResponse } from 'next/server'
+
+export async function GET(req: NextRequest) {
```

Comment:
HA! I think introducing a route where we let folks arbitrarily convert any URL of an SVG to a PNG is very risky!